src/components/User/PreferenceButton.jsx

```diff
@@ -0,0 +1,46 @@
+'use client';
+import React from 'react';
+import { Button, Typography, Box } from '@mui/material';
+import AddIcon from '@mui/icons-material/Add'; 
+
+const AddNewPreferenceButton = () => {
+  const handleClick = () => {
+    //change page to addPref page
+    console.log('New preference added!'); 
+  };
+
+  return (
+    <Box
+      sx={{
+        display: 'flex', 
+        justifyContent: 'center', 
+        alignItems: 'center', 
+        margin : 2,
+      }}
+    >
+      <Button
+        variant="contained"
+        onClick={handleClick} 
+        sx={{
+          bgcolor: 'text.secondary', 
```

Comment:
Could you please use `useStyles` for longer css snippets like this and elsewhere? 
https://mui.com/system/styles/api/#examples-2